process/predicting-milestones.md

```diff
@@ -166,13 +152,11 @@ Early testing
 
 Once all open issues are closed (or deferred to the next version) and the group believes the document is stable enough for wide review, a Working Draft can be published for wide review. Other Groups, expecially those with formal dependencies, are invired to review. External Groups with whom the Working group has an ongoing liaison should also be invited to review.
 
->
-Wide Review Duration
+> Time: Wide Review Duration
 : The minimum duration for the Wide Review comment period should be 4 weeks.
 {:.time}
 
->
-Wide Review Duration
+> Tips: Wide Review Duration
 : 'Horizontal' groups that review many specifications (for example, WAI P&amp;F and I18n Core) may, because of their workload, need longer than the minimum period.
```

Comment:
[WAI P&amp;F](https://www.w3.org/WAI/PF/) is defunct. This example should be replaced (and if an abbreviation or shortname is used, the long name should also be included).

```suggestion
: 'Horizontal' groups that review many specifications (for example, WAI P&amp;F, and [Internationalization Working Group (i18n-core)](https://www.w3.org/groups/wg/i18n-core/)) may, because of their workload, need longer than the minimum period.
```